docs/src/@primer/gatsby-theme-doctocat/nav.yml

```diff
@@ -123,3 +123,5 @@
     url: "/adr/0250-developing-and-publishing-clientside-behaviours"
   - title: 251. Use Catalyst for client-side behaviours
     url: "/adr/0251-use-catalyst-for-client-side-behaviours"
+  - title: 252. Build component css with ruby code
+    url: "/adr/0252-build-component-css-with-ruby-code"
```

Comment:
was this previously left out intentionally?

Reply:
I think it was accidentally left out and the build keeps adding it back